src/sentry/static/sentry/app/views/settings/projectAlerts/ruleDetailsNew/index.tsx

```diff
@@ -16,10 +16,17 @@ import RadioGroup from 'app/views/settings/components/forms/controls/radioGroup'
 import withOrganization from 'app/utils/withOrganization';
 import withProject from 'app/utils/withProject';
 
+type RouteParams = {
+  orgId: string;
+  projectId: string;
+  incidentRuleId: string;
+  ruleId: string; //TOOD: Make ruleId optional
```

Comment:
Another example where this field (`ruleId`) really is optional but breaks because of child components

Reply:
Can we change this to `TODO(ts)` so we can find it easier later?